data/core.yaml

```diff
@@ -785,6 +786,7 @@ en:
     best_imagery: Best known imagery source for this location
     global_imagery: This imagery source is available globally
     historic_imagery: This imagery is either considered outdated or there exists a more recent version from this provider
+    add_remove_favorite: Add or remove this imagery source to your list of favourites
```

Comment:
American English spelling, and I _think_ this preposition would be a little less awkward:

> Add or remove this imagery source <del>to</del> <ins>from</ins> your list of favo<del>u</del>rites

But consider splitting it up into two different messages, one for when clicking the icon would add and another for when it would remove.